locales/en-US/server.ftl

```diff
@@ -0,0 +1,161 @@
+homePageButtonMyShots = Go To My Shots
+homePageTeaser = Coming Soon...
+
+// leave screenshots page
+
+leavePageConfirmDelete = Confirm account deletion
+leavePageErrorAddonRequired = You must have the addon installed to delete your account
+// TODO: do we need this separate error message?
+leavePageErrorAddonRequired2 = You must have the addon installed to leave
+leavePageErrorGeneric = An error occurred
+leavePageWarning = This will permanently erase all of your Firefox Screenshots data.
+leavePageButtonProceed = Proceed
+leavePageButtonCancel = cancel
+leavePageDeleted = All of your screenshots have been erased!
+
+// not found page
+
+notFoundPageTitle = Page Not Found
+notFoundPageDescription = The page was not found.
+notFoundPageMyShotsLink = My Shots
+
+// shot page
+
+// In the following error messages, {status} is a placeholder for an HTTP status
+// code, like '500'. {statusText} is a text description of the status code, like
+// 'Internal server error'.
+shotPageAlertErrorUpdatingExpirationTime = Error saving expiration: {status} {statusText}
+shotPageAlertErrorDeletingShot = Error deleting shot: {status} {statusText}
+shotPageAlertErrorUpdatingTitle = Error saving title: {status} {statusText}
+
+shotPageShareButton
+  .title = Share
+shotPageCopy = Copy
+shotPageCopied = Copied
+shotPageShareFacebook
+  .title = Share to Facebook wall or message
+// TODO: awkward phrase, maybe 'Share on Twitter'?
+shotPageShareTwitter
+  .title = Share to a tweet
+shotPageSharePinterest
+  .title = Share to Pinterest
+shotPageShareEmail
+  .title = Create email with link
+shotPageShareLink = Get a shareable link to this shot:
+shotPagePrivacyMessage = This shot is only visible to you until you share the link.
+shotPageCopyImageText
+  .label = Copy Image Text
+shotPageConfirmDeletion = Are you sure you want to delete this shot permanently?
+// Note: {timediff} is a placeholder for a future time like 'tomorrow' or 'next week'
+// TODO: "in {timediff}" probably won't work across languages. maybe an absolute date is best.
+shotPageExpirationMessage = If you do nothing, this shot will be permanently deleted in {timediff}.
+// Note: {timediff} is a placeholder for a time duration phrase like '30 days'
+// TODO: again, "for {timediff}" may not work well.
+shotPageRestoreButton = restore for {timediff}
+shotPageExpiredMessage = This shot has expired.
+shotPageExpiredMessageDetails = Here is the page it was originally created from: {page}
+shotPageDeleteButton
+  .title = Delete this shot permanently
+shotPageAbuseButton
+  .title = Report this shot for abuse, spam, or other problems
+shotPageMyShots = My Shots
+shotPageDownloadShot
+  .title = Download the shot image
+shotPageDownload = Download
+// Note: do not translate 'Firefox Screenshots' when translating this string
+shotPageScreenshotsDescription  = Firefox Screenshots made simple. Take, save and share screenshots without leaving Firefox.
```

Comment:
Should this actually be "Firefox screenshots made easy", as in "Screenshots are now easy in Firefox"?